api/graphql/reservation_units/reservation_unit_serializers.py

```diff
@@ -217,7 +217,7 @@ class ReservationUnitCreateSerializer(ReservationUnitSerializer, PrimaryKeySeria
     allow_reservations_without_opening_hours = serializers.BooleanField(
         required=False,
         default=False,
-        help_text="Allow booking without opening hours. Used for testing.",
+        help_text="Allow reservations without opening hours. Used for testing.",
```

Comment:
This rename commit should be rebased into the original commit where this field was introduced to avoid code churn